src/components/RestrictedDocumentSelection/RestrictedDocumentSelection.tsx

```diff
@@ -129,9 +129,11 @@ export const RestrictedDocumentSelection = trackComponent(
             suggestCountries={suggestCountries}
             handleCountrySelect={handleCountrySelect}
             placeholder={translate(
-              'doc_select.section.input_placeholder_country'
+              'doc_select.section.input_placeholder_country_copy' // lokalise is weird and input_placeholder_country_copy is `Select issuing country`
```

Comment:
Any reason why this is "not working" as expected?

Reply:
i think it was manually input - I could change the key on lokalise but it requires to sync with IOS and Android
![image](https://user-images.githubusercontent.com/22033728/185448901-f321e86d-ec75-44e3-9365-e81171192f01.png)

Reply:
can't we just change the values? instead of the key? perhaps do the same across three SDKs.

Reply:
done - i created a ticket for Android and IOS does not have the issue